### ISSUE TITLE
Support for XCode version > 14

### ISSUE THREAD

**Issue by yorickreum** [nisargjhaveri/vscode-ios-debug#24]
Starting with XCode version 15, ios-deploy does not work properly anymore, causing this extension to crash. With XCode version 16 (currently beta) however, Apple introduced native LLDB support. See details in issue https://github.com/ios-control/ios-deploy/issues/588 Is there any plan on supporting XCode versions > 14 currently?

**Comment by nisargjhaveri** [comment]
Yes, I'd like to add the support. The last I had checked, there were still gaps in the devicectl and lldb to support the end to end scenarios. Seems like things might have changed in Xcode 16.

I'm hoping to get some time to work on this later this month. If you or someone else can help out with experimentation or changes, I'd be more than happy to review PRs in this direction.

**Comment by nisargjhaveri** [comment]
#18 also have some more conversations around this.
